Formula/opencv.rb

```diff
@@ -0,0 +1,157 @@
+class Opencv < Formula
+  desc "Open source computer vision library, version 3"
+  homepage "http://opencv.org/"
+
+  stable do
+    url "https://github.com/opencv/opencv/archive/3.2.0.tar.gz"
+    sha256 "b9d62dfffb8130d59d587627703d5f3e6252dce4a94c1955784998da7a39dd35"
+
+    resource "contrib" do
+      url "https://github.com/opencv/opencv_contrib/archive/3.2.0.tar.gz"
+      sha256 "1e2bb6c9a41c602904cc7df3f8fb8f98363a88ea564f2a087240483426bf8cbe"
+    end
+  end
+
+  head do
+    url "https://github.com/opencv/opencv.git"
+
+    resource "contrib" do
+      url "https://github.com/opencv/opencv_contrib.git"
+    end
+  end
+
+  option "with-contrib", 'Build "extra" contributed modules'
+  option "with-java", "Build with Java support"
+  option "with-tbb", "Enable parallel code in OpenCV using Intel TBB"
+  option "without-numpy", "Use a numpy you've installed yourself instead of a Homebrew-packaged numpy"
+  option "without-python", "Build without Python support"
+
+  option :cxx11
+
+  depends_on :ant => :build if build.with? "java"
+  depends_on "cmake" => :build
+  depends_on "pkg-config" => :build
+
+  depends_on "eigen"
+  depends_on "jpeg"
+  depends_on "libpng"
+  depends_on "libtiff"
+  depends_on "openexr"
+  depends_on :python => :recommended unless MacOS.version > :snow_leopard
+  depends_on "ffmpeg" => :optional
+  depends_on "tbb" => :optional
+  depends_on :java => :optional
+  depends_on :python3 => :optional
```

Comment:
There is only one PYTHONPATH; and opencv relies on it. Supporting Python 2 and 3 together broke people's build when using Python 3. This is why there is a check for that somewhere in the formula. But I would be willing to remove it and see what happens.